db/design_doc.go

```diff
@@ -348,7 +348,7 @@ func installViews(bucket base.Bucket) error {
 		                  	} else {
 		                       	sync = meta.xattrs.%s
 		                    }
-		                     `, base.SyncXattrName, base.SyncXattrName)
+		                     `, base.SyncPropertyName, base.SyncXattrName)
```

Comment:
Looks like both of these should be xattr, but we need to parameterise the `doc._sync` as well.